assets/styles/elements/button.css

```diff
@@ -16,3 +16,28 @@
     border: 0.0625rem solid var(--color-primary);
     color: var(--color-primary);
 }
+
+.button--round {
+    width: 3.75rem;
+    height: 3.75rem;
```

Comment:
> why are we using fixed width and height for the .button--round class instead of relying on padding?

Because of the aspect ratio, I guess. That is what I added now, plus padding.